### ISSUE TITLE
FileNotFoundError for chembl_filtered_vocabulary.txt When Running ppo.py

### ISSUE THREAD

**Issue by 11carlesnavarro** [Acellera/acegen-open#19]
Description
The script ppo.py fails to locate the chembl_filtered_vocabulary.txt file, resulting in a FileNotFoundError. This issue occurs even after resolving initial dependency-related errors.

Environment
```
Python Version: 3.10
Execution Path: /scratch/users/carles/repos/acegen-closed

```
Steps to Reproduce
Navigate to the repository's root: /scratch/users/carles/repos/acegen-open
Run the script: python scripts/ppo/ppo.py --config-name config_denovo

Expected Behavior
The script should successfully locate and read the chembl_filtered_vocabulary.txt file without errors.

Error Message
`FileNotFoundError: [Errno 2] No such file or directory: '/scratch/users/carles/mambaforge/envs/acegen/lib/python3.10/site-packages/acegen-1.0-py3.10.egg/priors/chembl_filtered_vocabulary.txt'
`

**Comment by MorganCThomas** [comment]
Can I check that you are installing acegen in develop mode? The files won't be available otherwise. 

I've made some improvements to packaging which should fix this #20

**Comment by MorganCThomas** [comment]
I believe this should be fixed now and behavior settled with #21
